.github/workflows/docker.yml

```diff
@@ -68,24 +71,25 @@ jobs:
         steps:
             - uses: actions/checkout@v3
             - name: Update Rust
+              if: env.SELF_HOSTED_RUNNERS == false
               run: rustup update stable
             - name: Dockerhub login
               run: |
                   echo "${DOCKER_PASSWORD}" | docker login --username ${DOCKER_USERNAME} --password-stdin
             - name: Cross build Lighthouse binary
+              if: env.SELF_HOSTED_RUNNERS == false
               run: |
                   cargo install cross
                   env CROSS_PROFILE=${{ matrix.profile }} CROSS_FEATURES=${{ matrix.features.env }} make build-${{ matrix.binary }}
+            - name: Make bin dir
+              if: env.SELF_HOSTED_RUNNERS == false
```

Comment:
Same with this, I'm not sure if we should just make this directory once on the server